server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/GroupItemNode.java

```diff
@@ -33,20 +37,50 @@
  */
 @Getter
 @ToString(callSuper = true)
-public class GroupItemNode extends VariableWithLevelNode {
+@EqualsAndHashCode(callSuper = true)
+public class GroupItemNode extends VariableWithLevelNode implements IUsageClause {
   private boolean redefines;
+  private UsageFormat usageFormat;
 
-  public GroupItemNode(Locality location, int level, String name, boolean global, boolean redefines) {
+  public GroupItemNode(
+      Locality location,
+      int level,
+      String name,
+      boolean global,
+      boolean redefines,
+      UsageFormat usageFormat) {
     super(location, level, name, VariableType.GROUP_ITEM, global);
     this.redefines = redefines;
+    this.usageFormat = usageFormat;
   }
 
   @Override
   public List<SyntaxError> process() {
     List<SyntaxError> errors = new ArrayList<>();
-    if (getChildren().stream().noneMatch(node -> node.getNodeType() == NodeType.VARIABLE))
+    if (isValidChildrenPresent()
+        && getChildren().stream().noneMatch(node -> node.getNodeType() == NodeType.VARIABLE))
```

Comment:
May this also be a part of the `isValidChildrenPresent()` method?

Reply:
done